elements/pf-date-picker/demo/pf-date-picker.html

```diff
@@ -0,0 +1,75 @@
+<link rel="stylesheet" href="demo.css">
+<script type="module" src="pf-date-picker.js"></script>
+
+<div class="container">
+  <section>
+    <h2>Basic</h2>
+    <pre><code>&lt;pf-date-picker&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <pf-date-picker></pf-date-picker>
+    <p>The basic date picker will use the user's locale to determine the date format. The default minimum valid date 
+      is <span>January 1, 1900</span>, and the default maximum valid date is <span>December 31, 9999</span>.</p>
+  </section>
+
+  <section>
+    <h2>Date Format</h2>
+    <pre><code>&lt;pf-date-picker dateFormatInput="YYYY-DD-MM"&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <pf-date-picker dateFormatInput="YYYY-DD-MM"></pf-date-picker>
```

Comment:
we should prefer dash-case attributes for camelCase DOM properties, i.e. `date-format-input`, since HTML attributes are case-insensitive.

Reply:
Changed the DOM properties format from cameCase to dash-case.